flask_project/campaign_manager/views.py

```diff
@@ -1039,48 +911,127 @@ def create_campaign():
         'create_campaign.html', form=form, **context)
 
 
+def get_edit_copy_arguments(campaign_obj):
```

Comment:
I feel we could find another name for the function get_edit_copy_arguments?
what does the function do? retrieve managers, get the campaign types, fill up a context hash and initialize a CampaignForm object. That's too much for a function.